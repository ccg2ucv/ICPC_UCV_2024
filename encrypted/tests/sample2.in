10
Mk2kiKd_ViMSOVYiY]M_\Yi;CC>i2yv
