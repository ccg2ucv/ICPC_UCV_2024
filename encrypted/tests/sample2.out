Casa_AZUL_CIELO_OSCURO_1994_sol
